ruby/vampires.rb

```diff
@@ -30,6 +32,7 @@ def vampire_test
 
 		puts "would you like to enroll in the company's health insurance? (y/n)"
 			wants_health_insurance = gets.chomp
+
 				if wants_health_insurance == "y"
 					wants_health_insurance = true
 				else 
```

Comment:
On line 66 consider this

I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.
```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```